### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Sboursen**
# Description

The final milestone of the BookstoreCMS project. I already styled it last week. So for this PR I just made it more responsive.

## Milestone requirements

- [x] Make the book cards responsive.
- [x] Make the header responsive.
- [x] Make the Form responsive.

## General requirements

- [x] There are no linter errors.
- [x] Followed the **Gitflow**.
- [x] The documentation is professional.

## JavaScript requirements

- [x] Followed the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

### this is a GIF for the new design after the first review.

bookstorecms

Thank you for taking the time to review this pull request.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,15 +7,11 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Web site created using create-react-app" />
-  <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
+  <meta property="og:title" content="Bookstore-CMS" />
```
Comment (Meltrust) [inline public/index.html:10]: - [ ] While your project is beautiful as it renders now (it is!), we need to match exactly each font styling for each element on the Zeplin.

## Explanation

Look:

Your Implementation | Zeplin Design
:---:|:---:
image | image

The font stylings in your project do not match the Zeplin.

We need to make your design an exact copy of the Zeplin design, this includes the font stylings.  The font stylings include: font-family, font-weight, line-height, letter-spacing, font-style, font-size, font-stretch, color.



## Hints

- Look:

image

- You will need to go through each element on the Zeplin and compare it to your implementation.  If it does not match, correct it as needed.

Look:

image


- You will need to import the RobotoSlab and Montserrat font families, each with the needed weights so they render correctly.

- I understand you used Tailwind for this project.  That framework should have the option to override its stylings with yours.

- This is a CSS project, please keep in mind the next reviewer will most likely check for CSS best practices on your overrides.

_Note:  These are just tips and examples intended to help.  The main requirement is to match your font stylings exactly as the Zeplin, as requested above._

That way, your project will look exactly like the Zeplin.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Sboursen,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Your implementation is beautiful✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: public/index.html (hunk 1 above)
Comment (Sboursen) [reply public/index.html:10]: Hi @Meltrust 

Thank you for the feedback 
As always, your reviews are very thorough. I appreciate that :+1:
I'll make sure to match the design as you requested. At least for large screens.

Best,

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Sboursen , 
# STATUS: APPROVED! 🥇 💯 
- [x] Good Job!! 👍 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**